tools/CMakeLists.txt

```diff
@@ -282,4 +292,9 @@ if(IREE_BUILD_COMPILER)
     DESTINATION "tests/bin"
     COMPONENT Tests
   )
-endif(IREE_BUILD_COMPILER)
+elseif(NOT IREE_HOST_BINARY_ROOT)
```

Comment:
It seems a bit weird that this is here instead of being part of the same conditional block as checking `IREE_HOST_BINARY_ROOT` above

Reply:
You can also blame/thank `IREE_ENABLE_THREADING` for that.

Ordering is currently:
* Import compiler binaries if needed
* Return if threading is disabled
* Define runtime binaries
* Define compiler binaries